src/config.ts

```diff
@@ -422,6 +423,9 @@ export const GAS_SCHEDULE_V1: FeeSchedule = {
     [ERC20BridgeSource.MultiHop]: fillData => {
         const firstHop = (fillData as MultiHopFillData).firstHopSource;
         const secondHop = (fillData as MultiHopFillData).secondHopSource;
+        if (!firstHop || !secondHop) {
```

Comment:
This code path will get called even if `MultiHop` samples nothing now (ie, it's disabled), so we need this catch.